stdlib/xml/dom/minidom.pyi

```diff
@@ -265,10 +265,10 @@ class Notation(Identified, Childless, Node):
     def __init__(self, name, publicId, systemId) -> None: ...
 
 class DOMImplementation(DOMImplementationLS):
-    def hasFeature(self, feature, version) -> bool: ...
-    def createDocument(self, namespaceURI: str | None, qualifiedName: str | None, doctype): ...
-    def createDocumentType(self, qualifiedName: str | None, publicId, systemId): ...
-    def getInterface(self, feature): ...
+    def hasFeature(self, feature: str, version: str | None) -> bool: ...
+    def createDocument(self, namespaceURI: str | None, qualifiedName: str | None, doctype: DocumentType | None) -> Document: ...
```

Comment:
I couldn't find definitive evidence for this in the source code, but the [documentation for this method](https://docs.python.org/3/library/xml.dom.html#xml.dom.DOMImplementation.createDocument) confirms this type annotation. Looks good.

Reply:
I had been trying to use https://cs.github.com/?scopeName=All+repos&scope=&q=repo%3Apython%2Fcpython+path%3ALib%2Fxml%2Fdom+%22.createDocument%28%22 to work backwards through what calls were being used and comparing it to the documentation as my way of trying to find evidence, if that helps

Reply:
Oh, that is helpful, thanks! Especially with `xml`, it's sometimes pretty difficult to figure out the types just by studying the source code, so hints as to how you figured out non-obvious types are definitely appreciated :)